salt/modules/yumpkg.py

```diff
@@ -1919,6 +1953,28 @@ def upgrade(name=None,
     return ret
 
 
+def update(name=None,
+            pkgs=None,
+            refresh=True,
+            skip_verify=False,
+            normalize=True,
+            minimal=False,
+            obsoletes=False,
+            **kwargs):
+    '''
+    Uses "upgrade" but defaults to obsoletes=False
+    Introduced to mirror expected CLI usage of yum update
+    For options see "upgrade"
```

Comment:
We can link to the docs for `upgrade()` to make things even better. Also, while we place `.. versionadded::` and `.. versionchanged::` inline to denote new arguments, for new _functions_ we tend to put this at the top before the function description, so that it appears right next to the function name in the docs. So, how about something like this.

```rst
    .. versionadded:: Fluorine

    Calls :py:func:`pkg.upgrade <salt.modules.yumpkg.upgrade>` with
    ``obsoletes=False``. Mirrors the CLI behavior of ``yum update``. See
    :py:func:`pkg.upgrade <salt.modules.yumpkg.upgrade>` for further
    documentation.
```